src/parser/diffeq_context.cpp

```diff
@@ -140,7 +140,7 @@ std::string DiffEqContext::get_cnexp_solution() const {
  * Return solution for euler method
  */
 std::string DiffEqContext::get_euler_solution() const {
```

Comment:
`nitpick`: still this method name `get_euler_solution` make sense here? the Euler solution now is mostly in `src/visitors/neuron_solve_visitor.cpp` at line `94-96` ?